modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java

```diff
@@ -360,15 +398,27 @@ public PNGImageWriteParam() {
     private int tileCacheDisabled = 0;
 
     /**
-     * Creates a new uninitialized builder for an {@linkplain #load image read}.
+     * Creates a new uninitialized builder for an {@linkplain #load image read} or
+     * a {@linkplain #mosaic mosaic operation}
      * 
-     * @see #load
+     * @see #load(String, int, boolean)
+     * @see #mosaic(RenderedImage[], MosaicType, PlanarImage[], ROI[], double[][], Range[])
      */
     public ImageWorker() {
         inheritanceStopPoint = this.image = null;
     }
 
     /**
+     * Creates a new uninitialized worker with RenderingHints for a {@linkplain #mosaic mosaic operation}
+     * 
+     * 
+     * @see #mosaic(RenderedImage[], MosaicType, PlanarImage[], ROI[], double[][], Range[])
+     */
+    public ImageWorker(RenderingHints hints) {
+        setRenderingHints(hints);
```

Comment:
Shouldn't this call the empty constructor? Or the empty constructor has useless code inside of it?